src/components/Form/CrossPodQueryForm.jsx

```diff
@@ -82,27 +97,48 @@ const CrossPodQueryForm = () => {
       statusType="Search status"
       defaultMessage="To be searched..."
     >
-      <form onSubmit={handleCrossPodQuery} autoComplete="off">
+      <Box sx={{ minWidth: 120 }}>
         <div style={formRowStyle}>
-          <label htmlFor="cross-search-doc">Search document from username: </label>
-          <br />
-          <br />
-          <input
+          <TextField
+            {...username}
+            label="Search document from username"
+            InputProps={{
+              type: 'search'
+            }}
             id="cross-search-doc"
-            size="60"
+            fullWidth
             name="crossPodQuery"
-            {...username}
             placeholder={selectedUser}
+            value={searchUser}
+            onChange={(event) => {
+              setSearchUser(event.target.value);
+            }}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="cross-search-doctype">Select document type to search: </label>
-          <DocumentSelection htmlId="cross-search-doctype" />{' '}
-          <button disabled={state.processing} type="submit">
+          <br />
+          <br />
+          <FormControl fullWidth autoComplete="off">
```

Comment:
`<FormControl>` is a `<form>` element, not a `<div>` or a `<label>`. Use `<FormControl>` to replace the `<form>`. This means Lines 102 to 118 should be inside `<FormControl>`